src/components/delete-account/delete-account-service.ts

```diff
@@ -16,21 +16,22 @@ export function deleteAccountService(
     sourceIp: string,
     sessionId: string,
     persistentSessionId: string,
-    clientSessionId: string
+    clientSessionId: string,
+    txmaAuditEncoded: string
   ): Promise<boolean> {
     const { status } = await axios.client.post<void>(
       API_ENDPOINTS.DELETE_ACCOUNT,
       {
         email: email,
       },
-      getRequestConfig(
+      getRequestConfig({
         token,
-        null,
```

Comment:
If this is removed then sourceIp would become validationStatuses parameter, is this correct?

Reply:
No worries, just noticed that getRequestConfig has been refactored down here. All good.